inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/common/EncryptUtil.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.manager.common.pojo.common;
+
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.crypto.Cipher;
+import javax.crypto.KeyGenerator;
+import javax.crypto.SecretKey;
+import java.io.BufferedInputStream;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.security.SecureRandom;
+import java.util.Properties;
+
+public class EncryptUtil {
+    private static final Logger logger =
+            LoggerFactory.getLogger(EncryptUtil.class);
+
+    private static final int KEY_SIZE = 128;
+
+    private static final String ALGORITHM = "AES";
+
+    private static final String CONFIG_FILE = "application.properties";
+
+    private static final String CONFIG_KEY = "inlong.encrypt.deskey";
+    private static final String RNG_ALGORITHM = "SHA1PRNG";
+
+    public static String DES_KEY;
+
+    /**
+     * Get deskey from config file
+     */
+    public static byte[] getDesKeyByConfig() throws Exception {
+        Properties properties = new Properties();
+        String path = Thread.currentThread().getContextClassLoader().getResource("").getPath() + CONFIG_FILE;
+        InputStream inputStream = new BufferedInputStream(Files.newInputStream(Paths.get(path)));
+        properties.load(inputStream);
+        String confDesKey = properties.getProperty(CONFIG_KEY);
+        if (StringUtils.isEmpty(confDesKey)) {
+            logger.error("cannot find deskey in property");
+            throw new Exception("cannot find deskey in property");
```

Comment:
why not use RuntimeException? Exception is too big

Reply:
fixed